web-services/query/src/main/java/datawave/webservice/query/configuration/GenericQueryConfiguration.java

```diff
@@ -47,6 +47,8 @@
     
     protected boolean bypassAccumulo;
     
+    protected boolean iteratorDebug = true;
```

Comment:
I think this should default to false

Reply:
I'm cool with defaulting to false, if there are no objections. But I'm curious to get your thoughts on DedupingIterator's `DEBUG` behavior in general.

The reason for defaulting to `true` was so that the existing behavior of DedupingIterator would be retained (assuming that's what we wanted), and to avoid the need for QLF.xml changes for defaulting to `true` there

And this new config setting was just to have a way of disabling it...mainly to prevent the `This event is 1 in 1Q!` error, which is something you're likely to run into on DW clusters that allow more than 500K results to be returned for a given query

So, if we default to false, I guess the question is, do we want to enable by default in QLF.xml or leave it disabled? If we leave it disabled, then perhaps the better question is, should we consider just getting rid of the `DEBUG` path inside DedupingIterator altogether and also this new config setting? That's the strategy I took with UniqueTransform in this PR, for the simple fact that the `true` path was never used and there was no way to enable it.

If this is something that will rarely be enabled in the future, then I think I'd much prefer to remove the debug behavior, and avoid adding yet another config setting.

Thoughts?